test-data/pybind11_mypy_demo/stubgen/pybind11_mypy_demo/basics.pyi

```diff
@@ -1,7 +1,7 @@
-from typing import ClassVar
+from typing import ClassVar, overload
 
-from typing import overload
 PI: float
+__version__: str
```

Comment:
Seems like we previous excluded this on purpose

Reply:
I support this change. I think I've seen several typeshed PRs where adding `__version__` to the stubs fixed false positives in user code (according to mypy_primer). I don't think there's a good reason to exclude `__version__` from the generated stubs.